editor-lsp/src/main/java/io/github/rosemoe/sora/lsp/editor/signature/SignatureHelpWindow.kt

```diff
@@ -125,10 +129,117 @@ open class SignatureHelpWindow(editor: CodeEditor) : EditorPopupWindow(
+            formatSignature(
+                signatures[i],
+                activeParameterIndex,
+                renderStringBuilder,
+                isCurrentSignature = i == activeSignatureIndex
+            )
+            if (i < activeSignatureIndex) {
+                renderStringBuilder.append("\n")
+            }
+        }
+
+        text.text = renderStringBuilder
+    }
+
+    private fun formatSignature(
+        signature: SignatureInformation,
+        activeParameterIndex: Int,
+        renderStringBuilder: SpannableStringBuilder,
+        isCurrentSignature: Boolean
+    ) {
+
+
+        val label = signature.label
+        val parameters = signature.parameters
+        val activeParameter = parameters.getOrNull(activeParameterIndex)
+
+        val parameterStart = label.substring(0, label.indexOf('('))
+        val currentIndex = 0.coerceAtLeast(renderStringBuilder.lastIndex);
+
+        renderStringBuilder.append(
+            parameterStart,
+            ForegroundColorSpan(defaultTextColor), SpannableStringBuilder.SPAN_EXCLUSIVE_EXCLUSIVE
+        )
+        renderStringBuilder.append(
+            "("
+        )
+
+        for (i in 0 until parameters.size) {
+            val parameter = parameters[i]
+            if (parameter == activeParameter && isCurrentSignature) {
+                renderStringBuilder.append(
+                    parameter.label.left,
+                    ForegroundColorSpan(highlightParameter),
+                    SpannableStringBuilder.SPAN_EXCLUSIVE_EXCLUSIVE
+                )
+                renderStringBuilder.setSpan(
+                    StyleSpan(Typeface.BOLD),
+                    renderStringBuilder.lastIndex - parameter.label.left.length,
+                    renderStringBuilder.lastIndex,
+                    SpannableStringBuilder.SPAN_EXCLUSIVE_EXCLUSIVE
+                )
+                if (i != parameters.size - 1) {
+                    renderStringBuilder.append(
+                        ", ", ForegroundColorSpan(highlightParameter),
+                        SpannableStringBuilder.SPAN_EXCLUSIVE_EXCLUSIVE
+                    )
+                }
+            } else {
+                renderStringBuilder.append(
+                    parameter.label.left,
+                    ForegroundColorSpan(defaultTextColor),
+                    SpannableStringBuilder.SPAN_EXCLUSIVE_EXCLUSIVE
+                )
+
+            }
+
+            if (i != parameters.size - 1 && (!isCurrentSignature || parameter != activeParameter)) {
+                renderStringBuilder.append(", ")
+            }
+
+        }
+
+        renderStringBuilder.append(")")
+
+        if (isCurrentSignature) {
+            renderStringBuilder.setSpan(
+                StyleSpan(Typeface.BOLD),
+                currentIndex,
+                renderStringBuilder.lastIndex,
+                SpannableStringBuilder.SPAN_EXCLUSIVE_EXCLUSIVE
+            )
+        }
 
     }
 
+
     private fun applyColorScheme() {
```

Comment:
Background of the tooltip window should be update as color scheme changes.
Consider adding a new color id in `EditorColorScheme`.